stream_alert_cli/test.py

```diff
@@ -16,219 +16,174 @@
 
 import base64
 import json
+import logging
 import os
-import sys
 
-BASEFOLDER = 'test/integration/fixtures'
+from main import StreamAlert
+from rules import (
+    sample_rules,
+    sample_matchers
+)
 
-def read_kinesis_records(local_directories):
-    """Read raw kinesis records and format for Lambda test event.
+SA_LOGGER = logging.getLogger('StreamAlert')
```

Comment:
Nit: When you have pairs like this, it's common to have the intent as the prefix. E.g.

LOGGER_SA
LOGGER_CLI
COLOR_RED
COLOR_GREEN
COLOR_RESET